src/amo/pages/Guides/index.js

```diff
@@ -0,0 +1,214 @@
+/* @flow */
+import Helmet from 'react-helmet';
+import * as React from 'react';
+import { compose } from 'redux';
+import { connect } from 'react-redux';
+
+import NotFound from 'amo/components/ErrorPage/NotFound';
+import { fetchGuidesAddons } from 'amo/reducers/guides';
+import { getAddonByGUID } from 'core/reducers/addons';
+import translate from 'core/i18n/translate';
+import { sanitizeHTML } from 'core/utils';
+import Icon from 'ui/components/Icon';
+import type { AddonType } from 'core/types/addons';
+import type { AppState } from 'amo/store';
+import type { I18nType } from 'core/types/i18n';
+import type { DispatchFunc } from 'core/types/redux';
+import type { ReactRouterMatchType } from 'core/types/router';
+
+import './styles.scss';
+
+type Props = {|
+  match: {
+    ...ReactRouterMatchType,
+    params: {| guidesSlug: string |},
+  },
+|};
+
+type InternalProps = {|
+  ...Props,
+  addons: Array<AddonType>,
+  clientApp: string,
+  dispatch: DispatchFunc,
+  guidesSlug: string,
+  guids: Array<string>,
+  i18n: I18nType,
+  lang: string,
+|};
+
+export const getGuids = (guidesSlug: string): Array<string> | null => {
+  switch (guidesSlug) {
+    case 'privacy':
+      return ['{446900e4-71c2-419f-a6a7-df9c091e268b}'];
+    // The following will be addressed in another issue.
+    // See https://github.com/mozilla/addons-frontend/issues/6744.
+    case 'organize-tabs-and-bookmarks':
+    case 'enhance-your-media-experience':
```

Comment:
I suppose you can delete this code for now (the two `case` above). There is a issue for that anyway.